build.fsx

```diff
@@ -67,8 +69,8 @@ Target "AssemblyInfo" (fun _ ->
   let shared =
       [ Attribute.Product project
         Attribute.Description summary
-        Attribute.Version release.AssemblyVersion
-        Attribute.FileVersion release.AssemblyVersion ] 
+        Attribute.Version version
```

Comment:
I think it's better to keep `release.AssemblyVersion` here. Assembly version doesn't have any effect in installing the extension.